composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "ac9e7012000affee34fbf8397ddc63f2",
+    "content-hash": "0940430285d0471a2f320eaefb26d64b",
```

Comment:
This file isn't really in scope for the PR, but is from running `composer update` after editing `composer.json`.